syntax = "proto3";

package greeter;

service Greeter {
  rpc GetIndexPage (EmptyRequest) returns (HtmlResponse);
}

message EmptyRequest {}

message HtmlResponse {
  string html_content = 1;
}
